src/components/paneset/Pane.js

```diff
@@ -93,19 +100,21 @@ export default class Pane extends Component {
       onExit,
       onExiting, // eslint-disable-line no-unused-vars
       onExited,
+      padContent,
       paneSub,
       paneTitle,
       paneTitleRef,
       static: isStatic,
       subheader,
+      tagName,
       visible,
       ...rest
     } = this.props;
     const {
       entered
     } = this.state;
 
-    let Element = aside ? 'aside' : 'section';
+    let Element = aside ? 'aside' : tagName;
```

Comment:
I think, `aside` prop is like a fifth wheel :) It's possible to assign `aside` tag through tagName, too. Is it possible to get rid of `aside` prop?

Reply:
Commit added that cleans that up a little. I dropped the `aside` prop and just use `tagName="aside"` instead.